ml-functions/MLFunctionsExample/ViewController.m

```diff
@@ -85,10 +85,10 @@ - (void)annotateImage:(NSDictionary *)requestData {
                             callWithObject:requestData
                                 completion:^(FIRHTTPSCallableResult * _Nullable result, NSError * _Nullable error) {
           if (error) {
-            if (error.domain == FIRFunctionsErrorDomain) {
+            if ([error.domain isEqualToString:@"com.firebase.functions"]) {
               FIRFunctionsErrorCode code = error.code;
               NSString *message = error.localizedDescription;
-              NSObject *details = error.userInfo[FIRFunctionsErrorDetailsKey];
+              NSObject *details = error.userInfo[@"details"];
```

Comment:
@paulb777 the functions constants don't seem to be importable into ObjC, so this file wouldn't build with the previous variable names.